texture_manager.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, fileObject=None):
         else:
             try:
                 self.textureData = json.load(
-                    fileObject, encoding='utf-8', cls=TextureConfigDecoder)
```

Comment:
You should probably add "encoding='utf-8'" to the open call of fileObject. Otherwise the file encoding used will be platform dependent.

Reply:
https://stackoverflow.com/questions/66032031/why-was-the-keyword-argument-encoding-removed-from-json-loads-in-python-3-9

Looks like json.load will detect the encoding automatically, hence specifying the encoding explicitly is not needed